README.md

```diff
@@ -1,14 +1,21 @@
-treeherder
+Treeherder
 ==================
 [![Build Status](https://travis-ci.org/mozilla/treeherder.png?branch=master)](https://travis-ci.org/mozilla/treeherder)
 [![Python Package Status](https://pyup.io/repos/github/mozilla/treeherder/shield.svg)](https://pyup.io/repos/github/mozilla/treeherder/)
 [![Node dependencies Status](https://david-dm.org/mozilla/treeherder/status.svg)](https://david-dm.org/mozilla/treeherder)
 [![Node devDependencies Status](https://david-dm.org/mozilla/treeherder/dev-status.svg)](https://david-dm.org/mozilla/treeherder?type=dev)
 [![Documentation Status](https://readthedocs.org/projects/treeherder/badge/?version=latest)](https://treeherder.readthedocs.io/?badge=latest)
 
+TABLE OF CONTENTS
+===================
+
+* [Description](#description)
+* [Instances](#instances)
+* [Installation](#installation)
+* [Links](#links)
 
 #### Description
-[Treeherder](https://treeherder.mozilla.org) is a reporting dashboard for Mozilla checkins. It allows users to see the results of automatic builds and their respective tests. The Treeherder service manages the etl layer for data ingestion, web services, and the data model behind Treeherder.
+[Treeherder](https://treeherder.mozilla.org) is a reporting dashboard for Mozilla checks. It allows users to see the results of automatic builds and their respective tests. The Treeherder service manages the etl layer for data ingestion, web services, and the data model behind Treeherder.
```

Comment:
"checkins" is actually more correct than "checks", since it's referring to people pushing code to repositories. Perhaps there's another way to phrase it that makes that clearer?